scrapinghub/client/jobs.py

```diff
@@ -422,35 +522,35 @@ class JobMeta(_MappingProxy):
 
     Usage::
```

Comment:
I'm a bit confused, why Usage is followed by `::`? When rst renders this will produce a literal block, I don't think we want literal block for the whole usage description, I expect that it's a list of short descriptions followed by a literal block with example:


```
Usage:

- get job metadata instance::

    >>> job.metadata
    <scrapinghub.client.jobs.JobMeta at 0x10494f198>
```